act-rules-format.bs

```diff
@@ -83,58 +83,43 @@ Even within a rules group, certain rules may not be used. This leaves one fewer
 To support users of ACT Rules in properly defining their test scenarios, ACT Rules SHOULD include a warning, if there are significant accessibility support concerns known about a rule.
 
 
-Test Subject Types (Input Data) {#input-types}
+Aspects Under Test {#input-aspects}
 ===================================
 
-Web pages, including publications and applications, go through many different stages before it is rendered to the end user. For example, PHP may be used to put various pieces of content into a template. That template is then sent as an HTML text document to a web browser. The browser in turn parses it and turns it into a DOM tree before rendering it to the screen. Accessibility tests could be run at each of these stages. It is therefore important to specify the *test subject type* that an ACT Rule expects.
+ACT Rules can operate on different aspects of the subject being tested. An aspect is a distinct part of the test subject itself or its underlying implementation. For example, rendering a web page to an end user involves multiple different technologies, some or all of which may be of interest to an ACT Rule. Some rules may need to operate directly on the HTTP messages exchanged between a server and a client, while others need to operate on the DOM tree exposed by a web browser. Some rules may even need to operate on several aspects simultaneously such as both the HTTP messages and the DOM tree.
 
-The following test subject types are common in accessibility testing:
-
-- HTTP Response
-- DOM Tree
-- Rendered Page
-- Template
-- Script
-
-Other types of test subjects may be possible. In those cases the ACT Rule MUST include a detailed description of the test subject type, or a reference to the that description.
+Each ACT Rule MUST include a description of all the aspects under test by the rule. Each aspect MUST be discrete and no overlap exist between aspects. Some aspects are already well defined within the context of web content, such as HTTP messages, DOM tree, and CSS styling, and do not warrant a detailed description. Other aspects may not be well defined or even specific to web content. In these cases, an ACT Rule SHOULD include a detailed description, or a reference to one, of the aspect in question.
 
+Common Aspects {#input-aspects-common}
+--------------------------------------
 
-HTTP Response Testing {#input-types-http-response}
---------------------------------------------
+### HTTP Messages ### {#input-aspects-http}
 
-Testing the files as they are served to the web browser (or other user agent) has its limitations. The files may be manipulated in different ways through presentation and scripting. Although this is an excellent place for parser testing.
+The HTTP messages exchanged between a client and a server as part of requesting a web page may be of interest to ACT Rules that wish to perform validation of e.g. HTTP headers or unparsed HTML and CSS.
```

Comment:
This sentence seems to be missing something or reads a little awkwardly.  Is it supposed to say, "...ACT Rules which perform validation, such as HTTP headers or unparsed HTML and CSS.

Reply:
of interest to ACT Rules. For example to perform validation of HTTP headers or unparsed HTML and CSS.